pkg/domain/common/configNames.go

```diff
@@ -3,17 +3,18 @@ package common
 import (
 	"errors"
 	"fmt"
+	"github.com/cloudogu/k8s-registry-lib/config"
 )
 
 type RegistryConfigKey interface {
-	GlobalConfigKey | DoguConfigKey | SensitiveDoguConfigKey
+	GlobalConfigKey | DoguConfigKey
```

Comment:
Since `SensitiveDoguConfigKey` is just a type alias for `DoguConfigKey`, I assume this matches it as well?

Reply:
yes! this code will be part of a bigger refactoring this week.